project_milestone/tests/test_project_milestone.py

```diff
@@ -0,0 +1,45 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestProjectMilestone(common.TransactionCase):
+
+    def test_create_data(self):

```

Comment:
Continuing with your free course on unit testing :grin:: Yes, sometimes you need to create test data, but most likely you can use the same data to perform several tests. In this case, change here:

```suggestion
    def setUp(self):
        super().setUp()
```

`setUp` is a method included in all `TransactionCase` subclasses, and it can be overriden to set up the data you need to run the tests in this case (class).

As a good rule of thumb, try to do all possible record creations in the `setUp`. You can assign `self.variable` to be able to access to it later in the actual test.